qa/resources/template.conf

```diff
@@ -30,6 +30,14 @@ scorex {
       wsServerPort = 0
   }
 
+  mempool {
```

Comment:
Check also the `template_predefined_genesis.conf` and its usage.
I think, we need to add the same there

Reply:
we have default values, no needed to add explicitly the conf there